docs/editor/accessibility.md

```diff
@@ -70,179 +70,181 @@ In `settings.json` file, nest the following code inside the outermost curly brac
 
-In the example below, the warning color is applied when a comma is missing after a JSON item.
+In the following example, the warning color is applied when a comma is missing after a JSON item.
 
-![JSON code to alter error/warning squiggle colors](images/accessibility/accessibility-extension-squiggles.png)
+![JSON code to alter colors for error and warning squiggles](images/accessibility/accessibility-extension-squiggles.png)
 
 * `editorError.foreground` - Overrides the wavy line beneath an error.
 * `editorWarning.foreground` - Overrides the wavy line beneath a warning.
 * `editorError.background` - Overrides the highlight color of an error.
 * `editorWarning.background` - Overrides the highlight color of a warning.
 
-Assigning a color to the background of `editorError` and `editorWarning` also helps to identify potential issues. The color that you choose will highlight the respective error or warning. The colors shown in the example above `#ffef0f` (yellow) and `#37777ff` (blue), are more accessible to individuals with common forms of color vision deficiencies.
+Assigning a color to the background of `editorError` and `editorWarning` also helps to identify potential issues. The color that you choose will highlight the respective error or warning. The colors shown in the preceding example, `#ffef0f` (yellow) and `#37777ff` (blue), are more accessible to individuals with common forms of color vision deficiencies.
 
 ### Selecting accessible colors
 
-The accessibility of colors is subjective to the type of anomalous trichromacy (color blindness). The level of severity ranges per person and can be divided into four condition types:
+The accessibility of colors is subjective to the type of anomalous trichromacy (colorblindness). The level of severity ranges per person and can be divided into four condition types:
 
 |   Condition      |   Type    |
 |       ---        |    ---    |
-|   Deuteranopia   |   Defined by the reduced sensitivity to green light. It is the most common form of color blindness.
-|   Protanopia     |   Defined by the reduced sensitivity to red light.
-|   Tritanopia     |   Defined by the reduced sensitivity to blue light. This condition is considered rare.
-|   Monochromia    |   Also referred to as, achromatopsia and is defined by the inability to see all colors. This is the rarest form of color blindness. Go to [Foundation for Fighting Blindness](https://www.fightingblindness.org/diseases/achromatopsia) for more information.
+|   Deuteranopia   |   Reduced sensitivity to green light. It is the most common form of color blindness.
+|   Protanopia     |   Reduced sensitivity to red light.
+|   Tritanopia     |   Reduced sensitivity to blue light. This condition is considered rare.
+|   Monochromia    |   The inability to see all colors, also referred to as *achromatopsia*. More information about the rarest form of colorblindness: [Foundation Fighting Blindness](https://www.fightingblindness.org/diseases/achromatopsia).
```

Comment:
italics doesn't render well on the website so we avoid using it for emphasis.
You can use bold for emphasis but not sure it's needed here